1.7/administration/upgrading.md

```diff
@@ -9,6 +9,7 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 
 **Important:**
 
+- This document is currently in BETA status. Due to a known issue, this install method will cause all apps and tasks to restart within your cluster.
```

Comment:
I'd like to add in that _I'm_ barely okay with this messaging. We'll be rolling out a 1.7.2 presently, and that will fix the issue. If this update doesn't release before 1.7.2, we probably want to omit that line.